src/styles/partials/components/_images.scss

```diff
@@ -0,0 +1,45 @@
+.dg_image-content {
+  display: block;
+  position: relative;
+  img {
+    display: block;
+    object-fit: fill;
+    width: 100%;
+  }
+  .dg_image {
+    display: block;
+    position: relative;
+  }
+}
+.left {
+  float: left;
+  margin-top: $margin-small;
+  margin-right: $padding-largest;
+  margin-bottom: $padding-largest;
+  max-width: 40%;
+}
+
+.right {
+  float: right;
+  margin-top: $margin-small;
+  margin-left: $padding-largest;
+  margin-bottom: $padding-largest;
+  max-width: 40%;
+}
+
+@media screen and (min-width: $large-breakpoint) {
+  .dg_image {
+    position: relative;
+    display: block;
+  }
+}
+
+@media screen and (max-width: $medium-breakpoint) {
+  .dg_image {
+    max-width: 100%;
+    position: relative;
+    display: block;
```

Comment:
lines 40 and 41 already declare the position and display, so it is redundant, as you move down the css file. These can be removed.

Reply:
done